distribution/docs/src/main/resources/_contents/_transformers/query-response-transformer-consumer-contents.adoc

```diff
@@ -0,0 +1,12 @@
+
+==== Query Response Transformer Consumer
+
+The Query Response Transformer Consumer is responsible for translating a {document} into a Catalog Metacard.
```

Comment:
it's an artifact from the template. It's meant to be replaced with the type of document. Fixed.